02Lesson/01Homework/index.html

```diff
@@ -19,7 +19,7 @@
           and deadlines under strict time constraints. In addition to my strong commitment
           and desire to succeed, I am able to work independently or as a team.</p>
    <h4>Experience</h4>
-        <p>I am a self taught and gradguate of the Austin Coding Academy
+        <p>I am a self taught and gradguate of the Austin Coding Academy</p>
```

Comment:
Hi Matt- a lot of your code looks good so far as I can see (as a noob)! My only recommendation is to consider spellcheck of content for projects you do plan on using for your future portfolio (for instance here- "graduate").